src/includes/configuration/before-send/java.spring.mdx

```diff
@@ -0,0 +1,32 @@
+`beforeSend` callback can be registered simply by creating a Spring bean implementing the `BeforeSendCallback`.
```

Comment:
We want to avoid suggesting how simple or not simple something is going to be in case the user doesn't have the same experience. 

```suggestion
`beforeSend` callback can be registered by creating a Spring bean implementing the `BeforeSendCallback`.
```

Reply:
Thanks!